### PR TITLE
release: firestore-export-bigquery

### PR DESCRIPTION
**Pull request by cabljac**
This PR is to release firestore-bigquery-export extension

### CODE REVIEW

**Review by pr-Mais — APPROVED** [review]

File: firestore-bigquery-export/extension.yaml [hunk 1]
```diff
@@ -48,6 +48,9 @@ roles:
   - role: datastore.user
     reason: Allows the extension to write updates to the database.
 
+  # - role: storage.objectAdmin
+  #   reason: Allows the extension to create objects in the storage bucket.
+
```
Comment (pr-Mais) [inline firestore-bigquery-export/extension.yaml:53]:
```suggestion
```

File: firestore-bigquery-export/extension.yaml [hunk 2]
```diff
@@ -491,6 +400,36 @@ params:
...
+  #   default: no
+  #   required: true
+
+  # - param: BACKUP_GCS_BUCKET
+  #   label: Backup GCS Bucket Name
+  #   description: >-
+  #     This (optional) parameter will allow you to specify a GCS bucket for which
+  #     failed BigQuery updates will be written to, if this feature is enabled.
+  #   type: string
+
```
Comment (pr-Mais) [inline firestore-bigquery-export/extension.yaml:429]:
```suggestion
```

File: firestore-bigquery-export/functions/stress_test/count.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+const admin = require("firebase-admin");
+
+// Initialize Firebase Admin with your credentials
+// Make sure you've already set up your Firebase Admin SDK
+admin.initializeApp({
+  projectId: "vertex-testing-1efc3",
```
Comment (pr-Mais) [inline firestore-bigquery-export/functions/stress_test/count.js:6]:
```suggestion
  projectId: "PROJECT_ID",
```

Comment (cabljac) [reply firestore-bigquery-export/functions/stress_test/count.js:6]: oop, will tidy this up in a new commit later

**Review by Tihana08 — APPROVED** [review]

**Pull request merged**